feature-libs/tracking/public_api.ts

```diff
@@ -1,4 +1,4 @@
 /*
  * Public API Surface of tracking
  */
-export {};
+export * from '@spartacus/tracking/personalization';
```

Comment:
We moved away from reexporting lower entrypoints from the main one/ 
```suggestion
export {};
```